default.yml

```diff
@@ -25,11 +25,6 @@ AllCops:
 Layout/LineLength:
   Max: 120
 
-# No space makes the method definition shorter and differentiates
-# from a regular assignment.
-Layout/SpaceAroundEqualsInParameterDefault:
-  EnforcedStyle: no_space
```

Comment:
I find the `no_space` option here feels cramped and is not one I am familiar with seeing in most ruby projects.


```ruby
# no_space
def my_method(working=true)
  ...
end

# default (space)
def my_method(working = true)
  ...
end
```

Reply:
I'm in the opposite camp here. A space feels odd to me in this context

Reply:
I agree with you @d3vkit and prefer the readability and consistency with the community on this one. However, at the end of the day, I am good either way.

Reply:
Same as [1]

Reply:
I'm in the @evan-waters camp here, but at the end of the day my main care is consistency. I think readability comes from consistency. So if we want to have spaces that's totally fine provided it's consistent throughout the codebase and moving forward, and it's not no spaces in some places and spaces in others.

Reply:
I prefer the space but I'm good either way since this is an auto-fix.